### PR TITLE
Added facebook and github buttons in SignUp page

### PR DESCRIPTION
**Pull request by JanviMahajan14**
This is how it looks like after the change
Screenshot from 2020-02-09 13-59-37

Pls have a look.

### CODE REVIEW

**Comment by JanviMahajan14** [comment]
@devesh-verma Please review this.

File: src/auth/signup-form/signup-form.js [hunk 1]
```diff
@@ -32,6 +32,19 @@ class SignUpForm extends Component {
             </Button>
           </div>
         </Form>
+        <div className="text-center text-muted signUp">Or sign up with</div>
+        <div className="text-center signUpButtons">
+          <Button type="submit" className="btn btn-primary">
```
Comment (vaibhavdaren) [inline src/auth/signup-form/signup-form.js]: make these button part of form class maybe.

Comment (JanviMahajan14) [reply src/auth/signup-form/signup-form.js]: Hi @vaibhavdaren  Did you mean that I should change the className sign to formSignup or something else?

**Comment by devesh-verma** [comment]
@JanviMahajan14 did you take the code from #new-development branch?
If so, make a new branch and send the PR to the development branch.

**Comment by devesh-verma** [comment]
you can close this PR.

**Pull request closed without merging**